src/server/game/Entities/Item/ItemTemplate.h

```diff
@@ -697,6 +697,13 @@ struct ItemTemplate
                GetSubClass() == ITEM_SUBCLASS_WEAPON_GUN ||
                GetSubClass() == ITEM_SUBCLASS_WEAPON_CROSSBOW;
     }
+    
+    bool IsRangedInventoryType() const
+    {
+        return InventoryType == INVTYPE_RANGED ||
```

Comment:
Another detail which makes the travis build error confusing, referring line 764, is that @VirusEntertainment is using @gegge's repo from 11. May this year. gegge's file is only 716 lines long, while the TC repo file has 771 lines.